source/includes/usage-examples/code-snippets/WatchCompanion.java

```diff
@@ -0,0 +1,34 @@
+package usage.examples;
+
+import java.util.Arrays;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.result.InsertOneResult;
+
+public class WatchCompanion {
+    public static void main(String[] args) {
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+            MongoCollection<Document> collection = database.getCollection("movies");
+
+            try {
+                InsertOneResult result = collection.insertOne(new Document("test", "sample movie document"));
```

Comment:
I just noticed that we're doing an insert, but specifying an option for full document on updates in the other app. Perhaps we could leave out the option in Watch.java or change this to add an update after the insert so the user can observe the defined behavior?